tests/unit/via/resources_test.py

```diff
@@ -8,18 +8,19 @@
 
 NORMALISED_URLS = [
     # URLs that should be returned unchanged
-    ("http://example.com", "http://example.com"),
-    ("https://example.com", "https://example.com"),
-    ("ftp://example.com", "ftp://example.com"),
-    ("http://example.com?a=1", "http://example.com?a=1"),
-    # URLs without a protocol that should have `https://` prefixed
-    ("example.com", "https://example.com"),
-    ("//example.com", "https://example.com"),
-    # Leading and trailing whitespace that should be stripped
-    (" http://example.com", "http://example.com"),
-    ("http://example.com ", "http://example.com"),
-    # Check we remove Via stuff
-    ("http://example.com?a=1&via.sec=TOKEN", "http://example.com?a=1"),
+    # ("http://example.com", "http://example.com"),
+    # ("https://example.com", "https://example.com"),
+    # ("ftp://example.com", "ftp://example.com"),
+    # ("http://example.com?a=1", "http://example.com?a=1"),
+    # # URLs without a protocol that should have `https://` prefixed
+    # ("example.com", "https://example.com"),
+    (".example.com", "https://example.com"),
+    # ("//example.com", "https://example.com"),
+    # # Leading and trailing whitespace that should be stripped
+    # (" http://example.com", "http://example.com"),
+    # ("http://example.com ", "http://example.com"),
+    # # Check we remove Via stuff
+    # ("http://example.com?a=1&via.sec=TOKEN", "http://example.com?a=1"),
```

Comment:
This was all commented probably testing locally

Reply:
Errr yup!

Reply:
This is now less comically broken.